token/core/common/authrorization.go

```diff
@@ -14,8 +14,10 @@ import (
 )
 
 type Authorization interface {
-	// IsMine returns true if the passed token is owned by an owner wallet in the passed TMS
-	IsMine(tok *token2.Token) ([]string, bool)
+	// IsMine returns true if the passed token is owned by an owner wallet.
+	// It returns the ID of the owner wallet and any additional owner identifier, if supported.
+	// It is possible that the wallet ID is empty and the additional owner identifier list is not.
+	IsMine(tok *token2.Token) (string, []string, bool)
```

Comment:
* Would it be a problem if we only passed the owner as a parameter and not the whole token?
* Maybe we could create a new struct
```
type TokenOwner struct {
 Owner view.Id
 AdditionalOwners []view.Id
}
```
and then define methods like `Equals` or `CanSpend` on this?

Reply:
It is important that the driver gets the entire token to take a proper decision